tests/testthat/helper-orderly.server.R

```diff
@@ -51,18 +51,6 @@ wait_for_finished_runner <- function(runner, key) {
 }
 
 
-test_runner <- function(path = tempfile()) {
```

Comment:
I thought I'd deleted this on 76f8197808512c0fa3ddf734a07118385964801c - where are we up to with integration?